CHANGES.rst

```diff
@@ -1,7 +1,9 @@
 0.1.3 (unreleased)
```

Comment:
Given the stable metadata below, I would consider bumping the version number to either 0.2 or even 1.0